pydantic/tools.py

```diff
@@ -77,3 +80,11 @@ def parse_raw_as(
         json_loads=json_loads,
     )
     return parse_obj_as(type_, obj, type_name=type_name)
+
+
+def schema(type_: Any, *, title: Optional[NameFactory] = None, **schema_kwargs: Any) -> 'DictStrAny':
```

Comment:
These methods looks useful, but they definitely deserve docstrings, and probably documentation too.

If it's too much for this PR, could we create an issue to remind us to document this in future?